webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -1855,3 +1855,97 @@ def mocked_supersearch_get(**params):
         response = self.client.get(url, {'product': 'foobaz'})
         eq_(response.status_code, 400)
         ok_('foobaz' in response.content)
+
+    @mock.patch('requests.get')
+    def test_SignatureSummary_exploitability(self, rget):
+
+        def mocked_get(url, params, **options):
```

Comment:
This is mocking requests.get which is called to do an HTTP GET from the middleware, right? Or is it for something else?